ansible/roles/mycar/files/my_joystick.py

```diff
@@ -121,31 +124,73 @@ def set_axis_rv(self, axis_val):
 
     def invert_controls(self):
         self.inverted = not self.inverted
+    
+    def go_easy_mode(self):
+        self.easy_drive_mode = not self.easy_drive_mode
+        self.init_trigger_maps()
+        
+    def left_shoulder_throttle(self):
+        self.set_throttle(-1)
+    
+    def right_shoulder_throttle(self):
+        self.set_throttle(1)
+    
+    def throttle_stop(self):
+        self.set_throttle(0)
 
     def init_trigger_maps(self):
         """
         init set of mapping from buttons to function calls
         """
-
-        self.button_down_trigger_map = {
-            'a_button': self.invert_controls,
-            #'b_button': self.toggle_manual_recording,
-            'x_button': self.x_button_pressed,
-            'y_button': self.emergency_stop,
-            'right_shoulder': self.increase_max_throttle,
-            'left_shoulder': self.decrease_max_throttle,
-            'options': self.toggle_constant_throttle,
-        }
-
-        self.axis_trigger_map = {
-            'left_stick_horz':  self.set_axis_lh,
-            'left_stick_vert':  self.set_axis_lv,
-            'right_stick_horz': self.set_axis_rh,
-            'right_stick_vert': self.set_axis_rv,
-            # Forza Mode
-            'right_trigger': self.magnitude(),
-            'left_trigger': self.magnitude(reversed = True),
-        }
+        if self.easy_drive_mode: 
```

Comment:
make the same thing with inverted control and remove if conditions on lh ... functions. Because they caused somes issues